### PR TITLE
Version fix 3759

### PR DESCRIPTION
**Pull request by ddonahue007**
I missed a few versions

### CODE REVIEW

File: packages/patternfly-4/react-topology/package.json [hunk 1]
```diff
@@ -27,7 +27,7 @@
   "dependencies": {
     "@patternfly/patternfly": "2.65.0",
     "@patternfly/react-core": "^3.140.6",
```
Comment (jschuler) [inline packages/patternfly-4/react-topology/package.json]: this one also?

Comment (jschuler) [reply packages/patternfly-4/react-topology/package.json]: actually i do see a prerelease with 3.140.6 in npm

File: packages/patternfly-4/react-inline-edit-extension/package.json [hunk 2]
```diff
@@ -28,8 +28,8 @@
   "homepage": "https://github.com/patternfly/patternfly-react/tree/master/packages/patternfly-4/",
   "dependencies": {
     "@patternfly/patternfly": "2.65.0",
-    "@patternfly/react-core": "^3.140.6",
```
Comment (evwilkin) [inline packages/patternfly-4/react-inline-edit-extension/package.json]: The latest prerelease for react-core is 3.140.6, so I believe this needs to stay as-is?

File: packages/patternfly-4/react-table/package.json [hunk 3]
```diff
@@ -28,8 +28,8 @@
   "homepage": "https://github.com/patternfly/patternfly-react/tree/master/packages/patternfly-4/react-table#readme",
   "dependencies": {
     "@patternfly/patternfly": "2.65.0",
-    "@patternfly/react-core": "^3.140.6",
```
Comment (evwilkin) [inline packages/patternfly-4/react-table/package.json]: The latest prerelease for react-core is 3.140.6, so I believe this needs to stay as-is?

File: packages/patternfly-4/react-topology/package.json (hunk 1 above)
Comment (ddonahue007) [reply packages/patternfly-4/react-topology/package.json]: Is this al set?  This is not a version i changed, it appears to be the latest pre-release though

Comment (evwilkin) [reply packages/patternfly-4/react-topology/package.json]: Yes I think this looks correct, the latest prerelease is 3.140.6 - the other places where this changes (react-inline-edit-extension/package.json and react-table/package.json) should be reverted back to match this 3.140.6 version number

**Pull request merged**